I need to install MKL manually before it becomes a dependency.

Hellos, 

I have my codes on github and I was expecting to run the following command without issues
```
import Pkg
Pkg.add(url="https://github.com/NoelAraujo/CoupledDipoles.jl")
```

I noticed that MKL.jl is the source of some problems, because somehow, it cannot be compiled (thus, my runtests fails on github actions)

_Sometimes_, the error is like this:
```
(...)
Building MKL → `/scratch/job.617411/input//.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/e01175ff53c062c52b6cbe441a4382e132f7c82e/build.log`
ERROR: LoadError: Error building `MKL`: 
 Downloading artifact: MKL
[ Info: Splicing in code to load MKL in /home/julia/bin/../share/julia/base/sysimg.jl
ERROR: LoadError: AssertionError: libblas_idx !== nothing && liblapack_idx !== nothing
Stacktrace:
 [1] replace_libblas(base_dir::String, name::String)
   @ Main /scratch/job.617411/input/.julia/packages/MKL/1aXoN/src/install.jl:10
 [2] change_blas_library(libblas::String)
   @ Main /scratch/job.617411/input/.julia/packages/MKL/1aXoN/src/install.jl:120
 [3] enable_mkl_startup()
   @ Main /scratch/job.617411/input/.julia/packages/MKL/1aXoN/src/install.jl:103
(...)
```
**However, here is the important part**: when I install MKL **first**, there is no error to install my codes
```
import Pkg
Pkg.add("MKL")
Pkg.add(url="https://github.com/NoelAraujo/CoupledDipoles.jl")
```

My best guess is that there is some racing condition with Julia multi-threading while compiling MKL.

Does anyone had a similar experience before?

--- a/.github/workflows/CompatHelper.yml
+++ b/.github/workflows/CompatHelper.yml
@@ -1,27 +1,24 @@
 name: CompatHelper
-
 on:
   schedule:
-    - cron: '0 1 * * *'
-  issues:
-    types: [opened, reopened]
-
+    - cron: '0 0 * * *'
+  push:
+      branches:
+        - 'main'
 jobs:
-  build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        julia-version: [1.3.1]
-        julia-arch: [x64]
-        os: [ubuntu-latest]
+  CompatHelper:
+    runs-on: ubuntu-latest
     steps:
-      - uses: julia-actions/setup-julia@latest
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v1
         with:
-          version: ${{ matrix.julia-version }}
-      - name: Install dependencies
-        run: julia -e 'using Pkg; Pkg.add(Pkg.PackageSpec(name = "CompatHelper", url = "https://github.com/bcbi/CompatHelper.jl.git"))'
+          version: '1'
+          arch: x64
+          show-versioninfo: true
+      - name: Pkg.add
+        run: julia --color=yes -e 'using Pkg; Pkg.add("CompatHelper")'
       - name: CompatHelper.main
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          JULIA_DEBUG: CompatHelper
-        run: julia -e 'using CompatHelper; CompatHelper.main()'
+          COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}
+        run: julia --color=yes -e 'using CompatHelper; CompatHelper.main(; include_jll = true)'

